soc/arm/nxp_lpc/lpc11u6x/soc.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, Seagate
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for nxp_lpc11u6x platform
+ *
+ * This module provides routines to initialize and support board-level
+ * hardware for the nxp_lpc11u6x platform.
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <soc.h>
+#include <zephyr/linker/sections.h>
+#include <zephyr/arch/cpu.h>
+#include <cortex_m/exc.h>
+
+/**
+ *
+ * @brief Perform basic hardware initialization
+ *
+ * Initialize the interrupt controller device drivers.
+ * Also initialize the timer device driver, if required.
+ *
+ * @return 0
+ */
+
+static int nxp_lpc11u6x_init(void)
+{
```

Comment:
why is this here and how this is related to this commit?

Reply:
This must be mistake occurred in rebase. I'll check them again.

Reply:
I just checked them again so this time should be ok.